README.md

```diff
@@ -97,12 +97,16 @@ $ NIX_BUILD_GROUP_NAME=nixbuilder ./nix-installer install linux-multi --nix-buil
 
 ### Upgrading Nix
 
-You can upgrade Nix (to the version specified [here](https://raw.githubusercontent.com/NixOS/nixpkgs/master/nixos/modules/installer/tools/nix-fallback-paths.nix)) by running:
+You can upgrade Nix by running:
 
 ```
 sudo -i nix upgrade-nix
 ```
 
+> **Note**
+> In `nix-installer` versions prior to v0.16.0, the version you will upgrade to is determined by `https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/installer/tools/nix-fallback-paths.nix`
+> In `nix-installer` version v0.16.0 and later, the version you will upgrade to is determined by [nix-upgrade]
```

Comment:
I don't think it's necessary to document old versions in this version's readme. The old versions are docunmented in their own readmes.

These sentences need periods at the end.

Reply:
I moved the `nix-upgrade` callout out of a note and into the actual "to upgrade to the version specified by nix-upgrade run:" preface.